---
string[] item_ids
string[] names
int32[] quantities
string[] warehouses
string[] racks
string[] cells
string[] statuses

